### ISSUE TITLE
An entire example

### ISSUE THREAD

**Issue by Hong-xiang-cv** [Duisterhof/rayst3r#2]
Hii,
    I just tried the app.py and eval.py，but get a different result from that reported in your paper and website. I don't know whether it goes wrong. If you could provide a entire example, it will help me a lot~

**Comment by Duisterhof** [comment]
Hi Hong-xiang-cv,

The results from the app.py should indeed be different, it uses Rembg and moge for the mask and depth.
eval.py should produce the same results as seen in the paper, can you share the results for executing:

`python3 eval_wrapper/eval.py example_scene/`

**Comment by Hong-xiang-cv** [comment]
Hi, Duisterhof

Thanks for your reply. I notice that there is no mask.png, rgb.png and depth.png in example_scene, maybe could you provide there example files in repo?

**Comment by Hong-xiang-cv** [comment]
Hi, Duisterhof

Thanks for your update. I run ```python3 eval_wrapper/eval.py example_scene/``` but the output point cloud is not complete. I wonder whether something is not appropriate.

<img width="448" alt="Image" src="https://github.com/user-attachments/assets/62ba103c-beae-44d1-8195-2563e21838ab" />

<img width="370" alt="Image" src="https://github.com/user-attachments/assets/c61ea7be-6875-4f02-b8a4-12e84121b148" />

**Comment by Duisterhof** [comment]
Hello, 

Thank you for your comments, and sorry for forgetting to upload the pngs.
This is the output I'm getting:
<img width="830" alt="Image" src="https://github.com/user-attachments/assets/87e384ed-52fe-45d1-937c-81b4f5a5d43a" />

<img width="830" alt="Image" src="https://github.com/user-attachments/assets/ba620c70-2c6c-405f-9041-127da9d88c63" />


However, when I do not compile curope, I'm getting this, with warning: "Warning, cannot find cuda-compiled version of RoPE2D, using a slow pytorch version instead"

<img width="377" alt="Image" src="https://github.com/user-attachments/assets/99586a4f-8db0-4232-b5be-1c2f64c4232a" />

I'll work on a fix, my guess is that it has to do with the mixed precision training and inference. 
Can you try with compiling curope?

**Comment by Hong-xiang-cv** [comment]
Hi, Duisterhof

Thanks very much for your reply and help! Problem solved!!

**Comment by Duisterhof** [comment]
Great! Thanks for opening an issue :)
